src/napari_imagej/settings.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+
+from dynaconf import Dynaconf
+
+# Preferences settings
+
+# The settings are in the root directory, three directories up
+SETTINGS_PATH = (Path(__file__).parent / "settings.toml").resolve()
```

Comment:
Will this do what you want if napari-imagej is installed via `pip install`? In that case, where *do* you want this `settings.toml` to live?

Reply:
Eh, the comment is incorrect now, because I moved the settings to the same directory as this file. I didn't want it right front and center in the repo if we aren't sure we will use it forever. Do you have opinions on where this should live?

Reply:
Yeah, I think it should live in the user's home directory, in `~/.config/napari/imagej.toml` or something. And if it doesn't exist, a default one should be written out. Because it won't necessarily be writable by the user—e.g. if it's installed system-wide. OTOH, I'm assuming that would be more work... there's not support for that sort of thing built in to dynaconf, is there? In practice most people use virtual environments they control, so it might be YAGNI in reality, but it would still be "more correct" to put the user's configuration in their configuration folder, rather than mashed in with the Python program.

Reply:
> OTOH, I'm assuming that would be more work... there's not support for that sort of thing built in to dynaconf, is there?

It *may* be, but it certainly isn't the [way they recommend doing it](https://www.dynaconf.com/#initialize-dynaconf-on-your-project)

Reply:
Alright, now our configuration file is stored there.